app/views/devise/shared/_links.html.erb

```diff
@@ -0,0 +1,21 @@
+<%- if controller_name != 'sessions' %>
+  <%= link_to "Log in", new_session_path(resource_name), class: "text-blue-800 underline" %><br />
+<% end %>
+
+<%- if devise_mapping.registerable? && controller_name != 'registrations' %>
+  <%= link_to "Sign up", new_registration_path(resource_name), class: "text-blue-800 underline" %><br />
+<% end %>
+
+<%- if devise_mapping.confirmable? && controller_name != 'confirmations' %>
+  <%= link_to "Didn't receive confirmation instructions?", new_confirmation_path(resource_name), class: "text-blue-800 underline" %><br />
+<% end %>
+
+<%- if devise_mapping.lockable? && resource_class.unlock_strategy_enabled?(:email) && controller_name != 'unlocks' %>
+  <%= link_to "Didn't receive unlock instructions?", new_unlock_path(resource_name), class: "text-blue-800 underline" %><br />
+<% end %>
+
+<%- if devise_mapping.omniauthable? %>
+  <%- resource_class.omniauth_providers.each do |provider| %>
+    <%= button_to "Sign in with #{OmniAuth::Utils.camelize(provider)}", omniauth_authorize_path(resource_name, provider), data: { turbo: false } %><br />
+  <% end %>
+<% end %>
```

Comment:
- [ ] According to the requirements:

![image](https://github.com/stephenkati/blog/assets/77069011/1b70adec-1afd-40cd-b20c-70ec015117b4)

A user should be able to reset the password. However, I'm not able to see the `Forgot password?` link. Please consider including the `:recoverable` logic in your project! 🤓 🔧